### PR TITLE
ign -> gz Header Migration : ign-utils (2)

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/ignition-tooling/release-tools/pull/712

Fixes a missing ignition header install.

### CODE REVIEW

**Comment by scpeters** [comment]
should this be rebased on `main`?

File: cli/include/CMakeLists.txt [hunk 1]
```diff
@@ -16,4 +16,6 @@ else()
 endif()
 
 add_subdirectory(gz/utils)
-install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
\ No newline at end of file
+install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+install(DIRECTORY external-cli/ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
+install(DIRECTORY vendored-cli/ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```
Comment (scpeters) [inline cli/include/CMakeLists.txt:21]: two problems:

* only one of these should be installed, never both, so this `install` logic needs to be inside the if/else block
* the `external-cli` and `vendored-cli` folders should never be part of the install path. I'm guessing we should add a `CMakeLists.txt` inside each subfolder that contains `install(DIRECTORY ignition ...` and `add_subdirectory(gz/utils)`

Comment (methylDragon) [reply cli/include/CMakeLists.txt]: Branch rebased, install logic fixed

https://github.com/ignitionrobotics/ign-utils/pull/55/commits/50f8dbde9bdf3fe5237a5845d6127e48c41a42e3

CI is green!

**Comment by scpeters** [comment]
I noticed two more issues with redirect headers:

* https://github.com/ignitionrobotics/ign-utils/pull/53#discussion_r861383740
* https://github.com/ignitionrobotics/ign-utils/pull/53#pullrequestreview-957145795

**Comment by scpeters** [comment]
I think this one can be squash-merged

**Review by scpeters — APPROVED** [review]

**Pull request merged**